.idea/misc.xml

```diff
@@ -6,6 +6,9 @@
         <entry key="..\:/Users/jorge/StudioProjects/imed-app/app/src/main/res/layout/activity_main.xml" value="0.2" />
```

Comment:
I think you should add this file (.idea/misc.xml) to git ignore.

Reply:
Thanks for the tip! Done and dusted
40cc91830cda85344a87d640ad61f0a7b23ecedb